src/modules/sdexec/sdexec.c

```diff
@@ -0,0 +1,850 @@
+/************************************************************\
+ * Copyright 2023 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+/* sdexec.c - run subprocesses under systemd as transient units
+ */
```

Comment:
super high level thought: split off libsubprocess rexec server into own module?  or into separate "thingy" (file in broker?) to parallel this module? ehhh maybe is overkill, but thought I'd just mention it.

Reply:
To clarify: did you mean moving `libsubprocess/server.[ch]` into `broker/exec.c`?

I think the plan is to reuse the subprocess rexec server in flux-shell someday so it probably should remain where it is.

I don't think there is a strong similar argument to be made for sdexec.  Instead of putting the service portion of the sdexec module into an internal convenience library, I put some useful "sdbus client" functions in libsdexec.

Reply:
> To clarify: did you mean moving libsubprocess/server.[ch] into broker/exec.c?

Yeah basically.  Wasn't sure if it was a good idea or not.  Definitely can put on hold for now.